README.md

```diff
@@ -72,6 +72,21 @@ By default, the application will use `mongoqp-cache/` within the system's
 temporary directory. This path, which must be writable, may be customized via
 the `twig.cache_dir` configuration option.
 
+#### Docker Setup 
+Clone the repo
+
+You just have to add the mongo uri in config.php.dist and run commands : 
+```
+$ docker build -t mongoqp .
```

Comment:
The [Configuration section](https://github.com/jmikola/mongoqp#configuration) above already mentions that `config.php.dist` can be copied to `config.php` and customized as needed. It seems reasonable that we can include the same directions here instead of suggesting that users modify `config.php.dist`.

That would also allow `COPY src/config.php.dist /mongoqp/src/config.php` to be removed from `Dockerfile` in https://github.com/jmikola/mongoqp/pull/21.